flocker/acceptance/test_api.py

```diff
@@ -32,13 +32,25 @@ def verify_socket(host, port):
     def can_connect():
         s = socket.socket()
         conn = s.connect_ex((host, port))
-        return False if conn else True
+        return conn == 0
 
     dl = loop_until(can_connect)
     return dl
 
 
-class ContainerAPITests(TestCase):
+class APITestCase(TestCase):
+    """
+    Base test case that cleans the cluster state before each test.
+    """
+    @require_flocker_cli
+    def setUp(self):
+        """
+        Reset the cluster to an empty deployment.
+        """
+        return get_nodes(self, num_nodes=2)
```

Comment:
The fact `get_nodes` cleans up nodes is not very clear... maybe rename `get_nodes` to `get_clean_nodes`?